src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -1241,6 +1241,7 @@ class FlowCrudSpec extends HealthCheckSpecification {
     @Tags([TOPOLOGY_DEPENDENT, LOW_PRIORITY])
     def "System allows to update single switch flow to multi switch flow"() {
         given: "A single switch flow with enabled lldp/arp on the dst side"
+        assumeTrue(useMultitable, "This test can be run in multiTable mode due to lldp/arp")
```

Comment:
maybe `can't`?

Reply:
This test is going to be run in case `multiTable` is enabled.
when `useMultiTable=false` then we will see the message that this test can be executed in multiTable mode.